tests/Dfe.PlanTech.AzureFunctions.UnitTests/Mappers/RecommendationChunkMapperTests.cs

```diff
@@ -146,7 +146,11 @@ public async Task Should_Update_Existing_Chunk()
         var csLink = new CSLink
         {
             Url = testUrl,
-            LinkText = linkText
+            LinkText = linkText,
+            Sys = new SystemDetails
+            {
+                Id = "csLink-id"
+            }
         };
 
         var payload = CreateRecommendationChunkPayload(
```

Comment:
This payload can then take just a csLinkId instead of a whole entity

Reply:
then on line 171/172, instead of existing assertions, you'd do `Assert.Equal(csLinkId, incoming.CSLinkId)`

The suggestion I'd made around db entities, I suggest not worrying about that after all, as that would warrant the user of `ValidateREferencedContent` which is for a list of references rather than 1